data/modules/scripts/gamestore/init.lua

```diff
@@ -402,7 +402,7 @@ function parseBuyStoreOffer(playerId, msg)
 
 	-- All guarding conditions under which the offer should not be processed must be included here
 	if
-		(table.contains(GameStore.OfferTypes, offer.type) == false) -- we've got an invalid offer type
+		(not table.contains(GameStore.OfferTypes, offer.type)) -- we've got an invalid offer type
```

Comment:
you can remove the parenthesis.
```suggestion
		not table.contains(GameStore.OfferTypes, offer.type) -- we've got an invalid offer type
```